common/views/pages/_app.tsx

```diff
@@ -158,7 +158,7 @@ const WecoApp: FunctionComponent<WecoAppProps> = ({
                   />
                   <LoadingIndicator />
 
-                  {civicUkApiKey && <CivicUK apiKey={civicUkApiKey} />}
+                  <CivicUK apiKey={civicUkApiKey || ''} />
```

Comment:
After https://github.com/wellcomecollection/wellcomecollection.org/pull/10905 I think this is safe to revert to:
```ts
{civicUkApiKey && <CivicUK apiKey={civicUkApiKey} />}
```

Reply:
Addressed in [latest commit](https://github.com/wellcomecollection/wellcomecollection.org/pull/10899/commits/b1531730b38e4d18835c6caab53e82757bccbf71) by merging contents of #10909